### DISCUSSION TITLE
Can multi-memory tier support more than 256GB cache memory

### DISCUSSION THREAD

**Discussion by tangliisu** [facebook/CacheLib#234] [Q&A]
According to some previous discussion(https://github.com/facebook/CacheLib/issues/193), we can't support more than 256GB cache memory. I saw there are some to-do on multi-memory tier support that each tier could have 128GB and we can create multiple tier (https://github.com/facebook/CacheLib/blob/94907cca05222c9d752181c4154c8008eb8a4e72/cachelib/allocator/memory/CompressedPtr.h#L46). does it means in multi-tier memory setting, we can have more than 256GB cache memory in future?

**Comment by haowu14** [comment]
Unfortunately no. We have a maximum of 2 tiers.

As you see in that discussion https://github.com/facebook/CacheLib/issues/193, the best way to address more memory is to increase the smallest addressable unit.

**Comment by pbhandar2** [comment]
Closing this discussion as it has been inactive. If this is still relevant, then please create a new discussion post and we will get back to you as soon as we can. Thank you!
